README.md

```diff
@@ -72,6 +72,7 @@ QGIS project files, documents, pictures and more. As soon as a project is upload
 - 🌈 **Web interface** - Simple user interface to view and manage projects
 - ⚡️ **Fast** - Efficient sync protocol transfering data between clients and server
 - 🧑‍💻 **Developer friendly** - Mergin is open platform. CLI tools and client libraries available for [Python](https://github.com/lutraconsulting/mergin-py-client) and [C++](https://github.com/lutraconsulting/mergin-cpp-client)
+- :camera: **Sync with database** - Supporting synchronisation of photos with common online cloud storage using [mergin-media-sync](https://github.com/lutraconsulting/mergin-media-sync) tool
```

Comment:
"Sync with database" should be probably changed to something else? (copy/paste?)